skyportal/models/shift.py

```diff
@@ -142,6 +142,15 @@ class Shift(Base):
         doc="Comments posted about this Shift.",
     )
 
+    reminders = relationship(
```

Comment:
One very cool thing that the reminders allow us to do is to do things like notify users when their shift is about to start. In general, i believe we could reuse this code to do the following later on: 
Add a reminder for all users 72 hours before a shift, that is triggered only if the shift is not full. That takes care of what we wanted to do which is notify users of a group if one of the shifts is not full yet. Same for 24 hours.
In general remind people that their shift is about to start. To do that we simply add a reminder at the same time as we add a ShiftUser, and remove it when we remove the shift user too.
Anyway, plenty of cool things can be done with this. Nice job !